app/config/Constants.scala

```diff
@@ -26,4 +26,6 @@ object Constants {
   val registrationConfirmationTemplateId = "oss_registration_confirmation"
 
   val schemeStartDate: LocalDate = LocalDate.of(2021, 7, 1)
+
+  val tradingNameReservedWords = Set("limited", "ltd", "llp", "plc")
```

Comment:
We also need to add "limited liability partnership" and "public limited company" (which means the implementation needs to change slightly)